v3/YALLStakingForeignMediator.md

```diff
@@ -83,6 +94,25 @@ CoolDownBox {
 }
 ```
 
+A delegate can unlock his stake with exclusion of the locked value. For ex. if Alice has 30K stake and 20K of this stake is locked, she could unstake only 10K.
+If she wants to unstake the entire stake of 30K, she should unlock it first, and only after unlocking she would be able unstaking the entier 30K stake.
+
+##### #lock(uint256 _amount)
+
+A delegate locks his stake
+
+* Requires that the current staked balance is greater than.
+* When locked, the current stake is not decremented.
+* The Foreign bridge notifies the Home bridge about locke balance update.
+
+##### #unlock(uint256 _amount)
```

Comment:
for current implementation it is impossible to unlock own stake. unlock will be implemented later in "verifierStakeSlasher" contract